README.md

```diff
@@ -210,7 +211,7 @@ Note: the Blazor Visual Studio extension is no longer required since .NET Core 3
 * [BlazorTypography](https://github.com/chanan/BlazorTypography) - ![last commit](https://img.shields.io/github/last-commit/chanan/BlazorTypography?style=flat-square&cacheSeconds=86400) A powerful toolkit for building websites with beautiful design ([Demo](https://chanan.github.io/BlazorTypography/)).
 * [Razor.SweetAlert2](https://github.com/Basaingeal/Razor.SweetAlert2) - ![last commit](https://img.shields.io/github/last-commit/Basaingeal/Razor.SweetAlert2?style=flat-square&cacheSeconds=86400) Blazor component implementing the popular SweetAlert2 JavaScript Library. 
 * [Blazor.SpinKit](https://github.com/faso/Faso.Blazor.SpinKit) - ![last commit](https://img.shields.io/github/last-commit/faso/Faso.Blazor.SpinKit?style=flat-square&cacheSeconds=86400) Blazor components for [SpinKit](https://tobiasahlin.com/spinkit/) spinners.
-* [Blorc.PatternFly](https://github.com/WildGums/Blorc.PatternFly) - ![last commit](https://img.shields.io/github/last-commit/WildGums/Blorc.PatternFly?style=flat-square&cacheSeconds=86400) Blazor wrappers for [PatternFly](https://www.patternfly.org). To view the latest develop branch in action, visit the [demo](http://blorc-patternfly.wildgums.com/) app.
+* [Blorc.PatternFly](https://github.com/WildGums/Blorc.PatternFly) - ![stars](https://img.shields.io/github/stars/WildGums/Blorc.PatternFly?style=flat-square&cacheSeconds=604800) ![last commit](https://img.shields.io/github/last-commit/WildGums/Blorc.PatternFly?style=flat-square&cacheSeconds=86400) Blazor wrappers for [PatternFly](https://www.patternfly.org). To view the latest develop branch in action, visit the [demo](http://blorc-patternfly.wildgums.com/) app.
```

Comment:
It looks like http://blorc-patternfly.wildgums.com/ is failing to load: "Failed to find a valid digest in the 'integrity' attribute for resource 'http://blorc-patternfly.wildgums.com/_framework/wasm/dotnet.3.2.0-rc1.20222.2.js' with computed SHA-256 integrity 'yVt8FYsTQDifOGsifIkmEXwe+7ML0jZ1dMi2xluiDXQ='. The resource has been blocked."

Reply:
> It looks like http://blorc-patternfly.wildgums.com/ is failing to load: "Failed to find a valid digest in the 'integrity' attribute for resource 'http://blorc-patternfly.wildgums.com/_framework/wasm/dotnet.3.2.0-rc1.20222.2.js' with computed SHA-256 integrity 'yVt8FYsTQDifOGsifIkmEXwe+7ML0jZ1dMi2xluiDXQ='. The resource has been blocked."

Thanks for reported it.

Reply:
http://blorc-patternfly.wildgums.com/ just loaded sucessfully.